pegasus/sites.v3/code.org/views/tutorial_promo.haml

```diff
@@ -8,7 +8,7 @@
   .col-25.specific-tutorial.left-tutorial{style: "display: #{tutorial[:display_second_tutorial]}"}
     .tutorial-box{id: tutorial[:id_for_mobile_image]}
       %h2.specific-tutorial-heading=tutorial[:left_heading]
-      .tutorial-info
```

Comment:
I made the change from `tutorial-info` to `tutorial-details` because there is conflicting styling in tools.css for `tutorial-info` related to something else.